cmd/di.go

```diff
@@ -88,13 +88,14 @@ func (di *Dependencies) Bootstrap(nodeOptions node.Options) error {
 
 	log.Infof("Starting Mysterium Node (%s)", metadata.VersionAsString())
 
-	if err := nodeOptions.Directories.Check(); err != nil {
-		return err
-	}
-
-	if err := nodeOptions.Openvpn.Check(); err != nil {
-		return err
-	}
+	// TODO Implement different check for in "mobile" package
```

Comment:
Maybe we can add check if it's mobile or not, to do checks for normal systems? 
These checks look important.

Reply:
This can be abstracted to some kind of Transport.Check() which can be conditionaly compiled on different oses.